### ISSUE TITLE
False positives of Alias Traversal

### ISSUE THREAD

**Issue by RicterZ** [yandex/gixy#88]
```
Description: Using alias in a prefixed location that doesn't ends with directory separator could lead to path traversal vulnerability. 
Additional info: https://github.com/yandex/gixy/blob/master/docs/en/plugins/aliastraversal.md
Pseudo config:

server {
	server_name xxx;

	location /bangumi {
		alias /www/bangumi;
	}

	location /ariang {
		alias /www/ariang;
	}
}
```
Which cannot perform a alias traversal attack.

**Comment by ss23** [comment]
Doesn't this have a similar issue in that if /www/bangumi-configuration existed, you could traverse to that directory? Less impact, but still valid

**Comment by buglloc** [comment]
@ss23 you're right. Therefore, Gisy will report this problem, but with less severity: https://github.com/yandex/gixy/blob/master/gixy/plugins/alias_traversal.py#L28

I will improve documentation for this case, most likely with a separate page :)

**Comment by brightball** [comment]
It's also reporting on direct file matches.

Should `location /favicon.ico` with an alias direct to the file be reported?

**Comment by brightball** [comment]
This also reports `/favicon.ico` with an alias pointing directly to the file as a vulnerability. Is that the case?

**Comment by ss23** [comment]
@brightball Without seeing your configuration, it's hard to tell exactly, but I believe it suffers from the same issue? That could match `/favicon.ico-secret-backup/database.sql` (though it's unlikely to exist).

Wouldn't you want to use `location = /favicon.ico` in your case?

**Comment by brightball** [comment]
The config that I'm looking at shows:

```		location /favicon.ico {
			alias /path/to/favicon.ico;
		}
```

I've seen this same approach used elsewhere for `manifest.json`, etc. Does `location = /favicon.ico` solve the problem?
